File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/interest-search-request.ts
import { IndustryClassEnum } from "projects/shared-models-lib/src/lib/enums/industry-class.enum";
import { InterestStatusEnum } from "../../shared/enum/interest-status.enum";
import { PagedRequest } from "projects/shared-models-lib/src/lib/pagination/PagedRequest";

export class InterestSearchRequest {
        industryClass: IndustryClassEnum;
        periodId: number;
        rolePlayerId: number;
        policyId: number;
        productCategoryId: number;
        interestStatus: InterestStatusEnum;

        pagedRequest: PagedRequest;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/InterestSearchRequest.cs

using RMA.Common.Entities.DatabaseQuery;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class InterestSearchRequest
    {
        public IndustryClassEnum? IndustryClass { get; set; }
        public int? PeriodId { get; set; }
        public int? RolePlayerId { get; set; }
        public int? PolicyId { get; set; }
        public int? ProductCategoryId { get; set; }
        public InterestStatusEnum? InterestStatus { get; set; }
        public PagedRequest PagedRequest { get; set; }
    }
}